CacheRefresh.php

```diff
@@ -2,53 +2,34 @@
 
 namespace SofaScore\Purgatory;
 
+use Exception;
+use RuntimeException;
 use SofaScore\Purgatory\Mapping\Loader\LoaderInterface;
 use SofaScore\Purgatory\Mapping\MappingCollection;
-use SofaScore\Purgatory\Mapping\MappingValue;
 use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
 
 class CacheRefresh
```

Comment:
Might as well rename the class when we're refactoring it. The class isn't actually doing anything related to Cache Refresh :).
Maybe we could just call it Purgatory as a main entry point?

Reply:
CacheRefresh is now renamed to Purgatory.